### ISSUE TITLE
Multiple emote icons have major readability/context issues.

### ISSUE THREAD

**Issue by moonheart08** [space-wizards/space-station-14#27861]
## funni
primary suspect:

![image](https://github.com/space-wizards/space-station-14/assets/7806367/508a920c-5e60-4377-aa65-2a6c0b0f8a2d)

this is the face of an insane man

## Description
Pretty much every icon modified from the scream icon (and honestly scream itself) looks... **eeeh**? They are, at least, not readable at first glance, with some like `crying` failing to express the intended emotion at all:
![image](https://github.com/space-wizards/space-station-14/assets/7806367/bd2ef84e-c35d-4804-9f50-82404b280418)
(headcanon: this is actually a smiling guy with facepaint)
## How Do We Fix This
My personal proposal is to completely ditch the idea of modeling emotes after a realistic face (which is entirely a consequence of them being edited tg mood icons.)
Instead, have simpler, stylized icons that can focus much harder on expressing the emotion within their limited pixel resolution and workspace. Anything recognizable at a glance, basically.

unironically icons like This (quickly thrown together in aseprite), which don't bother to be human at all and just use widely accepted ways to represent expressions like yelling:
![image](https://github.com/space-wizards/space-station-14/assets/7806367/45f4823b-b338-4949-a5e3-b2a73f49a9aa)

**Comment by Centronias** [comment]
I believe this should be considered resolved by [#30887](https://github.com/space-wizards/space-station-14/pull/30887).
